jena-arq/src/main/java/org/apache/jena/riot/RIOT.java

```diff
@@ -109,13 +109,6 @@ public static String getBuildDate() {
 
     // ---- Symbols
 
-    /**
-     * Symbol to use to pass (in a Context object) the "@context" to be used when reading jsonld
-     * (overriding the actual @context in the jsonld)
-     * Expected value: the value of the "@context",
-     * as expected by the JSONLD-java API (a Map) */
-    public static final Symbol JSONLD_CONTEXT = Symbol.create("http://jena.apache.org/riot/jsonld#JSONLD_CONTEXT");
-
```

Comment:
RIOT holds the public constants so an app does not need to import class `JsonLDReader` (which is in a subpackage).

It is possibly used by existing Jena applications so there is a compatibility issue here.

While not a perfect way to achieve a tidy separate of public and implementation I think we ought to keep the current style (and for `JSONLD_OPTONS`) unless there is a specific reason for this change.

Reply:
Same argument goes here: I was trying to build it consistent to the [JsonLdWriter](https://github.com/apache/jena/blob/master/jena-arq/src/main/java/org/apache/jena/riot/writer/JsonLDWriter.java#L99).

You can keep this one as well in RIOT. It is exactly the same Symbol. We could flag it as `@deprecated` if we want...

Reply:
As above, existing style is fine.

Leaving the constant, whether deprecated or not, helps migration.

(and I don't know why it is in RIOT in the first place.)